docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
@smarterclayton Just to clarify ... 
we want the following commands:
1) Specify username + password, optionally specify cacert and gitconfig
2) Specify ssh-private-key, optionally specify gitconfig
3) Specify gitconfig by itself, for which you can already do with ```oc secrets new```